apps/correlations/pkg/app/app.go

```diff
@@ -0,0 +1,52 @@
+package app
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-app-sdk/app"
+	"github.com/grafana/grafana-app-sdk/resource"
+	"github.com/grafana/grafana-app-sdk/simple"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog/v2"
+
+	correlationsv0alpha1 "github.com/grafana/grafana/apps/correlations/pkg/apis/correlation/v0alpha1"
+)
+
+func New(cfg app.Config) (app.App, error) {
+	simpleConfig := simple.AppConfig{
+		Name:       "correlation",
+		KubeConfig: cfg.KubeConfig,
+		InformerConfig: simple.AppInformerConfig{
+			ErrorHandler: func(ctx context.Context, err error) {
+				klog.ErrorS(err, "Informer processing error")
+			},
+		},
```

Comment:
FYI – this code is optional, @IfSentient correct me if I'm wrong. Nothing wrong with keeping it here though.

One small nit, however, is using `klog` – if you are keeping this code in, could you change it to use the app SDK logger:
```go
// Add an import
"github.com/grafana/grafana-app-sdk/logging"

// And use it!
ErrorHandler: func(ctx context.Context, err error) {
	logging.FromContext(ctx).Error("Informer processing error", "error", err)
},
```